babel.config.js

```diff
@@ -1,12 +1,13 @@
+// should track https://github.com/commercetools/ui-kit/blob/main/babel.config.js as suitable
 module.exports = {
   presets: [
     [
       '@commercetools-frontend/babel-preset-mc-app',
       {
-        // TODO: change this to `automatic` when using the new runtime
-        runtime: 'classic',
+        runtime: 'automatic',
```

Comment:
This should stay as `classic` for now. We can migrate to `automatic` as a separate step.

Reply:
I took this from the MC ui-kit to have the most similar setup. At least I hope I took the right piece. Set back to classic, but had no visible issues with automatic.